src/applications/vaos/new-appointment/sass/styles.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../../../node_modules/@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "../../../../../node_modules/@department-of-veterans-affairs/formation/sass/modules/m-process-list";
+@import "../../../../platform/forms/sass/m-schemaform";
+@import "src/applications/vaos/new-appointment/sass/calendar";
```

Comment:
I think you could do something like `@import "calendar";`, rather than using the whole path

Reply:
I think that was an ide ism

Reply:
Fixed